libafl/src/feedbacks/stdio.rs

```diff
@@ -15,7 +15,7 @@ use crate::{
     Error, HasMetadata,
 };
 
-/// Metadata for `StdOutFeddback`,
+/// Metadata for `StdOutToMetadataFeedback`.
```

Comment:
```rust
Metadata for [`StdOutToMetadataFeedback`].
```
would get us the link in the docs

(Does the metadata have to be `pub` at all?)

Reply:
Ah nevermind it needs to be pub if you want to read from it? Unless you got a function to access it

Reply:
Yes, I could `impl` a getter function for the field, but not a huge fan of that tbh